src/sass/partials/base/_content.scss

```diff
@@ -0,0 +1,70 @@
+@import 'content/variables';
+@import 'content/mixins';
+@import 'content/extends';
+
+* {
+  margin: 0;
+}
+
+html,
+body {
+  height: 100%;
+}
+
+body {
+  background-color: $white;
+  font-family: $font-stack;
+  margin: 0;
+}
+
+h1,
+h2 {
+  padding: .5em 0;
+}
+
+.wrapper {
+
+  @include breakpoint($small) {
+    max-width: 600px;
+  }
+
+  @include breakpoint($medium) {
+    max-width: 900px;
+  }
+
+  @include breakpoint($large) {
+    max-width: 1200px;
+  }
+
+  margin: 0 auto;
+  margin-bottom: -50px;
```

Comment:
to get the footer stuck to the bottom of the window, it's css tricks way. https://css-tricks.com/snippets/css/sticky-footer/

I can change it if you know of a better method